caramel-domain/src/test/kotlin/com/whatever/caramel/domain/notification/service/ScheduledNotificationServiceTest.kt

```diff
@@ -0,0 +1,110 @@
+package com.whatever.caramel.domain.notification.service
+
+import com.whatever.caramel.common.util.toDateTime
+import com.whatever.caramel.domain.notification.model.NotificationType
+import com.whatever.caramel.domain.notification.model.ScheduledNotification
+import com.whatever.caramel.domain.notification.repository.ScheduledNotificationRepository
+import com.whatever.caramel.domain.notification.vo.NotificationMessageVo
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.time.LocalDate
+import java.time.LocalDateTime
+import kotlin.test.Test
+
+class ScheduledNotificationServiceTest {
+    private lateinit var mockScheduledNotificationRepository: ScheduledNotificationRepository
+    private lateinit var service: ScheduledNotificationService
+
+    @BeforeEach
+    fun setUp() {
+        mockScheduledNotificationRepository = mockk(relaxed = true)
+        service = ScheduledNotificationService(mockScheduledNotificationRepository)
+    }
+
+    @DisplayName("알림 메시지가 주어졌을 때, ScheduledNotification 객체로 변환하여 저장을 요청한다")
+    @Test
+    fun scheduleNotifications() {
+        // given
+        val myId = 1L
+        val partnerId = 2L
+        val message = NotificationMessageVo(NotificationType.ANNIVERSARY_YEARLY, "1주년", "축하해요!")
+        val messagesByUserId = mapOf(myId to message, partnerId to message)
+        val notifyAt = LocalDate.of(2025, 8, 8).toDateTime()
+
+        // when
+        service.scheduleNotifications(messagesByUserId, notifyAt)
+
+        // then
+        verify(exactly = 1) { mockScheduledNotificationRepository.saveAll(any<List<ScheduledNotification>>()) }
```

Comment:
```suggestion
        verify(exactly = 1) { mockScheduledNotificationRepository.saveAll(any<List<ScheduledNotification>>()) }
```
요기에 eq로 들어갈 값을 검증해줘도 좋을듯 합니당~

Reply:
밑에 모두다 !

Reply:
앗 이걸 이제봤네요 빠르게 수정하고 올리겠습니다

Reply:
반영 완료 했습니다~
`verify(exactly = 0)` 같은 경우는 `any()`로 넣어두는게 더 넓은 범위로 테스트 할 수 있어서 그대로 두었습니다!